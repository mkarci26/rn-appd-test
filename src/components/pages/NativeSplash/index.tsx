import { ReactElement } from 'react';
import React = require('react');

const NativeSplash = (): ReactElement => {
  return (
    <>Landing Screen</>
  );
};

export default NativeSplash;
