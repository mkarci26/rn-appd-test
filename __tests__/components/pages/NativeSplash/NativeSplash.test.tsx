import 'react-native';
import React from 'react';
import { render, waitFor } from '@testing-library/react-native';
import NativeSplash from 'components/pages/NativeSplash';
const props: any = {
  some: 'some'
}

describe('NativeSplash test suite', () => {
  
  it('Page to be loaded', async () => {
    const { toJSON, getByTestId } = render(
      <NativeSplash {...props} />
    );
    await waitFor(() => {
      expect(toJSON()).toMatchSnapshot();
    });
  });
});