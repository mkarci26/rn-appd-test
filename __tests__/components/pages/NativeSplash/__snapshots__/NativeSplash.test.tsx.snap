// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NativeSplash test suite Page to be loaded 1`] = `"Landing Screen"`;
